common/lib/xmodule/xmodule/tests/test_video.py

```diff
@@ -42,6 +101,7 @@ def instantiate_descriptor(**field_data):
 
 
 class VideoModuleTest(LogicTest):
+
```

Comment:
Well... the pep8 plugin for Sublime is what adds them - sorry I was unsure as to what introduced them, but basically I had the plugin reformat the file to fix the problem, and that is what adds the extra line.

Shame because it's a very useful plugin, keeping things clean - and there does not appear to be an option to not add those lines. (I suppose the authors of the login at least think that it is somehow more pep8y to have them!). I'll remove them - maybe I can edit the plugin source...